README.md

```diff
@@ -363,5 +363,13 @@ We use `yarn` package manager and `node 18`.
 - Use `yarn test` to run tests on all the `*.test.ts` files.
 - Use `yarn lint` to run the linter.
 - Use `yarn fix` to fix all the auto fixable issues reported by the linter.
+
 ## Deployment
-Pending on https://github.com/paritytech/review-bot/issues/55
+To deploy a new version you need to update two files:
+- [`package.json`](./package.json): Update the version number.
+- [`action.yml`](./action.yml): Update the image number in `runs.image`.
+**Important**: Both versions must have the same number.
```

Comment:
it could be done with some release script? 

[yarn version](https://yarnpkg.com/cli/version) which should update package.json
but action could be done with awk + regex?

Reply:
It can be done. I just didn't think that investing the time on writing a script to replace two strings is worth the effort as the change is minimal. But I can look into adding some script to do that.

Or would it be fine if I move it into a different ticket?

Reply:
sure :)